gsw/stability.py

```diff
@@ -47,9 +47,12 @@ def Nsquared(SA, CT, p, lat=None, axis=0):
     Returns
     -------
     N2 : array
-        Buoyancy frequency-squared at pressure midpoints, 1/s.
+        Buoyancy frequency-squared at pressure midpoints, rad^2/s^2.
         The shape along the pressure axis dimension is one
         less than that of the inputs.
+        The units of N2 are radians2 s-2 however in may textbooks this is
```

Comment:
I would leave out this block. Surely the audience for gsw can be expected to know the difference between radians and cycles. Specification of s^{-2} as the units for N^2 is not an abbreviation and is not restricted to textbooks; it is standard.  Where cycles are desired as the unit of phase, it is explicit, e.g., Hz, or cycles per hour.   But if you really want to be more explicit here, I would change the "rad^2/s^2" in the top changeset line to "s^{-2}", and then say, "(Frequency N is in radians per second.)", and leave it at that.